raft_test.go

```diff
@@ -2337,17 +2337,71 @@ func TestRaft_LeadershipTransferWithOneNode(t *testing.T) {
 	}
 }
 
+func TestRaft_LeadershipTransferWithWrites(t *testing.T) {
```

Comment:
I couldn't give you numbers, but it didn't take many tries to provoke a failure.  It can still fail if I up the concurrency enough, e.g. if I run 16 parallel -race instances.  But I suspect the same is true for many other tests, as the failures are things like timeouts due to heartbeats not being processed quickly enough.